pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.mypy]
+check_untyped_defs = true
+disallow_any_generics = true
+disallow_incomplete_defs = true
+disallow_untyped_calls = false
+disallow_untyped_defs = true
+no_implicit_optional = true
+show_error_codes = true
+strict_equality = true
+strict_optional = true
+warn_redundant_casts = true
+warn_unreachable = true
+warn_unused_configs = true
+warn_unused_ignores = true
+
+[[tool.mypy.overrides]]
+module = [
+    "androguard.*",
+    "uplink",
+    "requests.*",
+    "authlib.*",
+    "pytest",
+    "pyaxmlparser.*"
+]
```

Comment:
Ooops, these modules are related to another project!
I think you can remove that section and see if the last step of `pipenv run lint` (see other comment) complains about missing imports. If it does, add that modules to the list.

Reply:
Removed the modules and works